data/locale/sv.ts

```diff
@@ -450,18 +450,18 @@ If you&apos;re interested in translating LMMS in another language or want to imp
     </message>
     <message>
         <source>Automation Editor - no pattern</source>
-        <translation>Redigera Automation - inget automationsmönster</translation>
+        <translation>Automatisk redigerare - inget mönster</translation>
```

Comment:
Once again, this isn't an "automatic editor", it's an editor for automation patterns (which currently don't have a good translation for)